web-api/migration-terraform/main/lambdas/migrations/0009-remove-blocked-cases-from-eligible-for-trial-record.js

```diff
@@ -0,0 +1,28 @@
+const migrateItems = async (items, documentClient) => {
+  const itemsAfter = [];
+  for (const item of items) {
+    if (item.pk === 'eligible-for-trial-case-catalog') {
+      const caseRecord = await documentClient
+        .get({
+          Key: {
+            pk: `case|${item.docketNumber}`,
+            sk: `case|${item.docketNumber}`,
+          },
+          TableName: process.env.SOURCE_TABLE,
+        })
+        .promise()
+        .then(res => {
+          return res.Item;
+        });
+
+      if (!caseRecord.automaticBlocked && !caseRecord.blocked) {
```

Comment:
it would seem that a previous migration is altering this data before it is saved into the new location. does the caseRecord need a check for whether or not it's calendared?

Reply:
Good catch! Calendared cases will never have eligible for trial records. If they did, we'd have much bigger problems than this migration.